packages/expo-store-review/ios/StoreReviewModule.swift

```diff
@@ -21,4 +21,16 @@ public class StoreReviewModule: Module {
       }
     }.runOnQueue(DispatchQueue.main)
   }
+
+  private func isRunningFromTestFlight() -> Bool {
+    #if targetEnvironment(simulator)
+    return false
+    #endif
+
+    return Bundle.main.appStoreReceiptURL?.lastPathComponent == "sandboxReceipt" && !hasEmbeddedMobileProvision()
```

Comment:
may be worth adding a comment with an explanation and reference for future travellers, one about the sandboxReceipt but and then also

> Apps distributed through TestFlight or the App Store will not have an embedded provisioning profile. Source: https://developer.apple.com/documentation/technotes/tn3125-inside-code-signing-provisioning-profiles#Profile-location